src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -2170,10 +2170,10 @@ WelcomeToadlet.restartNode=Restart Freenet
 WelcomeToadlet.restarting=Please wait a moment while your Freenet node is being restarted.  This screen should refresh automatically.  Thank you for using Freenet.
 WelcomeToadlet.restartingTitle=The Freenet node is being restarted.
 WelcomeToadlet.searchBoxLabel=Search Freenet (experimental)
-WelcomeToadlet.searchBoxWarningSlow=Searching can be slow and unreliable, so this will open a new window. You should have a look at the bookmarks while you wait for the search. WARNING: This search, like the rest of Freenet, is not filtered, and could find offensive or illegal content. Be careful!
+WelcomeToadlet.searchBoxWarningSlow=Will open in new window. Can be slow and unreliable. Take a look at the bookmarks while you wait. WARNING: This search, like the rest of Freenet, is not filtered, and could return offensive or illegal content. Be careful!
```

Comment:
We're not bound to twitter message length, we might as well include articles where appropriate. → "Search will open in **a** new window."